### DISCUSSION TITLE
AG 1.2.0 (BETA 3) the FX plugin work on ABLETON LIVE 11 on Mac OSX CATALINA ?

### DISCUSSION THREAD

**Discussion by Rikyild** [apohl79/audiogridder#553] [Help]
with the new beta the FX plugin does not open on Ableton live 11. while on the other Daw it's ok (Logic, studio one), the midi plugin work better. someone else has this problem or it's just my problem? I had to go back to the stable version. thanks for your help.

**Comment by CaBuffalo** [comment]
No, we don't hear any feedback like this.

Do you get any error messages? Can you be more specific?

Thanks.

On Sat, May 15, 2021 at 6:35 AM Rikyild ***@***.***> wrote:

> with the new beta the FX plugin does not open on Ableton live 11. while on
> the other Daw it's ok (Logic, studio one), the midi plugin work better.
> someone else has this problem or it's just my problem? I had to go back to
> the stable version. thanks for your help.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/553>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6GCFE57ZPVNE6VQGADTNZ2BZANCNFSM4454WRKQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Rikyild** [reply]
Many thanks for your answer, no any message , no crash, it just doesn't open. 
It's really weird, with all the versions I tried it never happened.
by reinstalling the version 1.1.1 will work normally again!

**Reply by apohl79** [reply]
I recently tried the current ableton live demo version and ag was loading and showing the UI.

If you test 1.2 again, please enable tracing and logging in the plugin and share the trace and log file of the session. https://audiogridder.com/bug-report-diagnostics-locations/

**Comment by CaBuffalo** [comment]
If you can, uninstall 1.1.1 and then do a clean1.2 Beta3. See what happens.

On Sat, May 15, 2021 at 8:39 AM Rikyild ***@***.***> wrote:

> Many thanks for your answer, no any message , no crash, it just doesn't
> open.
> It's really weird, with all the versions I tried it never happened.
> by reinstalling the version 1.1.1 will work normally again!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/553#discussioncomment-742156>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CRQV4TDS7ISXKF5MLTN2IUDANCNFSM4454WRKQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Rikyild** [reply]
I tryed... but no resolve,same issue, I also cancelled the cache and all remaininig files but nothing resolve, I tried on 2 different machines with different OS, (Imac catalina and mac book pro High sierra) same result, attached the video.
everything is ok on Win10 on another machine. 

https://user-images.githubusercontent.com/79607805/118371251-02265100-b5ac-11eb-84d4-2bc205f613cb.MOV

it is definitely a problem related to the control of ableton plugins, everythins is ok on the other DAW like Logic and studio one.

This happens with both Live 10 and 11.

Many thanks for your help
